packages/lambda-tiler/src/routes/import.ts

```diff
@@ -0,0 +1,63 @@
+import { HttpHeader, LambdaHttpRequest, LambdaHttpResponse } from '@linzjs/lambda';
+import { Config } from '@basemaps/shared';
+import { createHash } from 'crypto';
+import { findImagery, RoleRegister } from '../import/imagery.find.js';
+import { Nztm2000Tms, TileMatrixSets } from '@basemaps/geo';
+import { makeCog } from '../import/make.cog.js';
+import { ConfigProcessingJob } from '@basemaps/config';
+import { CogJobFactory } from '@basemaps/cli';
+
+/**
+ * Trigger import imagery job by this endpoint
+ *
+ * @example
+ * - /v1/import?path=s3://linz-imagery-staging/2022-03/wellington_rural_2022_delivery_1
+ */
+export async function Import(req: LambdaHttpRequest): Promise<LambdaHttpResponse> {
+  const path = req.query.get('path');
+  const projection = req.query.get('p');
+
+  // Parse projection as target, default to process both NZTM2000Quad
+  let target = Nztm2000Tms;
```

Comment:
I think there is a function that does this for us?
```
getTileMatrix(randomSTring)
```

possibly in @basemaps/shared

Reply:
const `targetTileMatrix = getTIleMatrix(projection) ?? Nztm2000Tms`

Reply:
I think we need to keep the logic here to return 404 if projection url parameter is incorrect. In case someone put a wrong projection code but lead to default 2193.